stable/kong/README.md

```diff
@@ -363,7 +363,7 @@ The custom resources are:
 
 You can can learn about kong ingress custom resource definitions here:
```

Comment:
Can we instead hyperlink with the text "here"?

Meaning, make "here" link to the link below?

Reply:
done in https://github.com/helm/charts/pull/14606/commits/c4d3bd39928d09cd5b71bde89fc0eefbd0b2d5c6